packages/broker/src/ConfigWizard.ts

```diff
@@ -82,200 +84,164 @@ const DefaultConfig: Config = {
             }
         },
-    }
+    },
 }
 
-export class ConfigWizard{
-    config: Config
-    // required to mock results on tests
-    inquirer: inquirer.Inquirer = inquirer
-
-    constructor() {
-        this.config = DefaultConfig
-    }
-
-    private async inquirerSinglePrompt(prompt: inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion) {
-        const answers = await this.inquirer.prompt([prompt])
-        return answers[prompt.name!]
-    }
-
-    private async promptNumberWithDefault(prompt: inquirer.InputQuestion, defaultValue: number): Promise<number>{
-        const valueString = await this.inquirerSinglePrompt(prompt)
-        const value = valueString === '' ? defaultValue: parseInt(valueString)
-
-        if (Number.isNaN(value) || !Number.isInteger(value)) {
-            logger.warn(`Non-numeric value [${valueString}] provided`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
-        }
-
-        if (value < 1024 || value > 49151) {
-            logger.warn(`Out of range port [${value}] provided (valid range 1024-49151)`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
-        }
-
-        return value
-    }
-
-    private async promptExistingPathWithDefault(prompt: inquirer.InputQuestion, defaultValue: string): Promise<string>{
-        const returnedValue = await this.inquirerSinglePrompt(prompt)
-        // defaultValue always exists at this point
-        if (returnedValue === '') {
-            return defaultValue
-        }
-
-        if (!existsSync(returnedValue)) {
-            logger.warn(`Path [${returnedValue}] does not exist`)
-            return await this.promptExistingPathWithDefault(prompt, defaultValue)
-        }
-
-        return returnedValue
-    }
-
-    async generateOrImportPrivateKey(): Promise<Wallet>{
-        const generateOrImport = await this.inquirerSinglePrompt({
-            type: 'list',
-            name:'generateOrImportEthereumPrivateKey',
-            message: 'Do you want to generate a new Ethereum private key or import an existing one?',
-            choices: ['Generate', 'Import'],
-            filter: (choice: string) => {
-                return choice.toLowerCase()
-            }
-        })
-
-        let wallet: Wallet
-        if (generateOrImport === 'generate') {
-            wallet = Wallet.createRandom()
-            this.config.ethereumPrivateKey = wallet.privateKey
-            logger.info('This is your generated private key. Please store it in a secure location')
-            logger.alert(this.config.ethereumPrivateKey)
-        } else if (generateOrImport === 'import') {
-            const privateKey = await this.inquirerSinglePrompt({
-                type: 'input',
-                name: 'privateKey',
-                message: "'Please provide the private key to import'",
-            })
-
+export let prompts: Array<inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion> = [
```

Comment:
Use `const` instead of `let` (there can be a temporary non-exported value to evaluate the prompts value, but export it as a const)